qnap/shared/teliod.cgi

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
+echo "Content-Type: application/json"
+echo ""
+
+# Varibles set by the HTTP web server
+#
+# $REQUEST_METHOD
+# $QUERY_STRING
+# $CONTENT_LENGTH
+
+POST_DATA=""
+
+if [[ "$REQUEST_METHOD" == "POST" || "$REQUEST_METHOD" == "PATCH" ]]; then
+    read -n $CONTENT_LENGTH POST_DATA
+fi
+
+QPKG_DIR="/share/CACHEDEV1_DATA/.qpkg/NordSecurityMeshnet"
+APP_TMP_DIR="/tmp/nordsecuritymeshnet"
+APP_CMD="$QPKG_DIR/teliod"
+CONFIG_FILE="$QPKG_DIR/teliod.cfg"
+LOG_FILE="$APP_TMP_DIR/teliod.log"
+PID_FILE="$APP_TMP_DIR/teliod.pid"
+
+is_application_running() {
+    if [ -f ${PID_FILE} ]; then
+        PID=$(cat ${PID_FILE})
+        if [ -d /proc/${PID}/ ]; then
+            echo 1
```

Comment:
nit: it's abit weird, that we return 1 on success, while in `start_application` for other success we check if return is 0. But not sure on how to proceed with this, since my logic wants to see 1 if running, but also, wants to see 0 if success (meaning it's running)... So not really an action point, but just a comment for thinking

Reply:
That also crossed my mind, what would make sense here, if we're running a bash command normally is how you described, but the return value for `is_application_running()` made sense to me to be true/false.

Reply:
whatever you decide, good for me, just wanted to point out :smile: